articles/quickstart/webapp/python/01-login.md

```diff
@@ -18,186 +18,208 @@ github:
 
 <%= include('../../../_includes/_logout_url', { returnTo: 'http://localhost:3000' }) %>
 
-## Configure Flask to Use Auth0
+## Install dependencies
 
-### Add the Dependencies
+For the purposes of this example, we'll be using the [Authlib](https://authlib.org/) OAuth library and [Flask](https://flask.palletsprojects.com/en/2.0.x/).
 
-This example uses [Flask](http://flask.pocoo.org) and the [Authlib](https://github.com/lepture/authlib) OAuth library.
+Begin by creating a `requirements.txt` file in your project directory:
 
-Add the following dependencies to your `requirements.txt` and run `pip install -r requirements.txt`.
+```python
+# 📁 requirements.txt -----
+
+flask>=2.0.3
+python-dotenv>=0.19.2
+authlib>=1.0
+requests>=2.27.1
+```
+
+You should now run `pip install -r requirements.txt` from your shell to make these dependencies available to your project.
+
+## Configure your `.env` file
+
+Next, create an `.env` file in your project directory. This file will hold your client keys and other configuration details.
 
-```js
-// /requirements.txt
+```ini
+# 📁 .env -----
+
+AUTH0_CLIENT_ID=${account.clientId}
+AUTH0_DOMAIN=${account.namespace}
+AUTH0_AUDIENCE=
```

Comment:
If you specify an audience, you're going to need a client secret to authenticate the code exchange.

I would just drop the audience bit (like the express one does https://auth0.com/docs/quickstart/webapp/express)